rf-stylez.gemspec

```diff
@@ -1,35 +1,38 @@
 # coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
+# frozen_string_literal: true
+
+lib = File.expand_path("../lib", __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'rf/stylez/version'
+require "rf/stylez/version"
 
 Gem::Specification.new do |spec|
-  spec.name          = 'rf-stylez'
+  spec.name          = "rf-stylez"
   spec.version       = Rf::Stylez::VERSION
-  spec.authors       = ['Emanuel Evans']
-  spec.email         = ['emanuel@rainforestqa.com']
-  spec.license       = 'MIT'
+  spec.authors       = ["Emanuel Evans"]
+  spec.email         = ["emanuel@rainforestqa.com"]
```

Comment:
We might want to change this as I think that email goes nowhere. Don't know where we'd want it to go though. 🤷